### PR TITLE
add extended follow up to prevax

### PR DESCRIPTION
**Pull request by kurttaylor**
add extended follow up analyses to the prevax cohort - adapted from [this PR](https://github.com/opensafely/post-covid-pre-vaccinated-cardiovascular/pull/14)

### CODE REVIEW

File: analysis/active_analyses.R [hunk 1]
```diff
@@ -58,12 +58,17 @@ outcomes <- c("type 1 diabetes",
...
 
 outcome_group <- "diabetes"
 
-outcomes_short <- c("t1dm","t2dm", "t2dm_rec", "t2dm_pre_rec", "t2dm_post_rec", "t2dm_follow", "t2dm_pd","t2dm_pd_no", "t2dm_obes","t2dm_obes_no", "otherdm","gestationaldm")
-outcome_venn <- c(TRUE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE)
+outcomes_short <- c("t1dm","t2dm", "t2dm_rec", "t2dm_pre_rec", "t2dm_post_rec", "t2dm_follow", "t2dm_pd","t2dm_pd_no", "t2dm_obes","t2dm_obes_no", "otherdm","gestationaldm",
+                    "t1dm_extended_follow_up","t2dm_extended_follow_up", "otherdm_extended_follow_up","gestationaldm_extended_follow_up")
+outcome_venn <- c(TRUE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE)
 
 for (i in 1:length(outcomes)) {
```
Comment (RochelleKnight) [inline analysis/active_analyses.R:73]: You're setting covid_history to TRUE for the extended follow up analyses - do you want this?

**Review by RochelleKnight — CHANGES_REQUESTED** [review]

**Comment by RochelleKnight** [comment]
I think the only bit you're missing is some of the bits where the variable follow_up_end_exposure_period  is used/I don't think you create it in the end dates script

**Comment by kurttaylor** [comment]
Thanks @RochelleKnight  - I have added these now

File: analysis/active_analyses.R (hunk 1 above)
Comment (kurttaylor) [reply analysis/active_analyses.R:73]: I've turned all COVID history analyses off now as none of them have ran anyway

**Comment by kurttaylor** [comment]
Reverted changes after discussing with @RochelleKnight via Slack. Ready to merge.

**Pull request merged**